src/main/java/com/xpanxion/java/assignments/student2/Worker2.java

```diff
@@ -1,8 +1,25 @@
 package com.xpanxion.java.assignments.student2;
 
+import com.xpanxion.java.assignments.DataAccess;
+import com.xpanxion.java.assignments.model.Product;
+
+import java.util.HashMap;
+import java.util.List;
+
 public class Worker2 {
 
+    DataAccess dataAccess = new DataAccess();
+    HashMap<Integer, String> depNames = new HashMap<>();
+
     public void ex1() {
+        depNames.put(1,"Electronics");
```

Comment:
Let's try to convert the list of department names from DataAccess to a Map:  https://howtodoinjava.com/java8/collect-stream-to-map/